### PR TITLE
Update the profile STIG for OL9

### PR DESCRIPTION
**Pull request by mrkanon**
#### Description:

STIG profiles for OL9 are updated based on preliminary DISA requirements

#### Rationale:

This is a draft set variables and rules within the profile to better align with DISA STIG draft for OL9

### CODE REVIEW

File: linux_os/guide/services/sssd/sssd_offline_cred_expiration/oval/shared.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <criteria operator="OR">
       <criterion comment="Check offline_credentials_expiration in /etc/sssd/sssd.conf"
       test_ref="test_sssd_offline_cred_expiration" />
-      {{% if product in ["ol8", "rhel8"] %}}
+      {{% if product in ["ol8", "ol9", "rhel8", "rhel9"] %}}
```
Comment (Xeicker) [inline linux_os/guide/services/sssd/sssd_offline_cred_expiration/oval/shared.xml:7]: Could someone of @ComplianceAsCode/red-hatters validate this is also applicable to rhel9

It seems like it should be from here: https://www.stigviewer.com/stig/red_hat_enterprise_linux_9/2023-12-01/finding/V-258133

**Review by Xeicker — APPROVED** [review]

File: linux_os/guide/services/sssd/sssd_offline_cred_expiration/oval/shared.xml (hunk 1 above)
Comment (marcusburghardt) [reply linux_os/guide/services/sssd/sssd_offline_cred_expiration/oval/shared.xml:7]: I can confirm. Thanks

**Review by marcusburghardt — APPROVED** [review]

**Comment by marcusburghardt** [comment]
@Xeicker the author sent a force-push after the approval. We have to review the whole PR again before merging it.

@mrkanon whenever possible, avoid overwriting existing commits after a review. It is fine to create an additional commit with the new changes. It also makes the review process easier.
In some cases it could be reasonable to rework an existing commit and force-pushing it. In this case, make it clear the reason and what was changed. Thanks for the contributions! :+1:

**Comment by Xeicker** [comment]
@marcusburghardt Hi, sure I'll check it complete. This force push was due to an internal discussions, as the commits included some unresolved conflicts, and to avoid backporting that into our scap-security-guide package.

We'll try to avoid this, and of course commenting the why in case we can't avoid it

**Comment by marcusburghardt** [comment]
All good @mrkanon and @Xeicker . Thanks for the contributions and the information. :+1:

**Comment by Xeicker** [comment]
The errors are not in the scope of this MR so I'll merge it

The errors are:

```
ERROR - Environment failed to prepare, skipping test
```
 &
```
ERROR - Rule evaluation resulted in error, instead of expected fixed during remediation stage 
ERROR - The remediation failed for rule 'xccdf_org.ssgproject.content_rule_file_ownership_audit_configuration'.
```

**Pull request merged**